src/accessors/external_accessor.ts

```diff
@@ -873,6 +873,26 @@ export class ExternalAccessor implements IManagementApiAccessor, IManagementSock
     return httpResponse.result;
   }
 
+  public async getTokensForFlowNodeByProcessInstanceId(
+    identity: IIdentity,
+    processInstanceId: string,
+    flowNodeId: string,
+  ): Promise<DataModels.TokenHistory.TokenHistoryGroup> {
+
+    const requestAuthHeaders: IRequestOptions = this._createRequestAuthHeaders(identity);
+
+    const restPath: string = restSettings.paths.getTokensForFlowNodeByInstanceId
```

Comment:
Almost missed it. Sorry 😉 

```suggestion
    const restPath: string = restSettings.paths.getTokensForFlowNodeByProcessInstanceId
```